### PR TITLE
auto-deploy CParser docker container

### PR DESCRIPTION
**Pull request by lsf37**
This action triggers docker container deployment in the repo [seL4/ci-actions](https://github.com/seL4/ci-actions) when the C parser changes here.

### CODE REVIEW

File: .github/workflows/trigger.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+jobs:
+  trigger:
+    name: CParser Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.SSRG_BAMBOO_REPO }}
+          repository: seL4/ci-actions
+          event-type: cparser-deploy
```
Comment (mbrcknl) [inline .github/workflows/trigger.yml:27]: I couldn't find a workflow to receive this event. I guess you still need to add that?

**Review by mbrcknl — APPROVED** [review]
Looks good!

File: .github/workflows/trigger.yml (hunk 1 above)
Comment (mbrcknl) [reply .github/workflows/trigger.yml:27]: Never mind. I should have looked in the [pull requests](https://github.com/seL4/ci-actions/pull/106)!

Comment (lsf37) [reply .github/workflows/trigger.yml:27]: I also just added that after this one, so there was a race condition :-)

**Pull request merged**